app/javascript/packages/document-capture/components/in-person-prepare-step.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  Alert,
+  Button,
+  IconList,
+  IconListItem,
+  PageHeading,
+  ProcessList,
+  ProcessListItem,
+} from '@18f/identity-components';
+import { removeUnloadProtection } from '@18f/identity-url';
+import { useContext } from 'react';
+import { FlowContext } from '@18f/identity-verify-flow';
+import { useI18n } from '@18f/identity-react-i18n';
+import InPersonTroubleshootingOptions from './in-person-troubleshooting-options';
+
+function InPersonPrepareStep() {
+  const { t } = useI18n();
+  const { inPersonURL } = useContext(FlowContext);
+
+  return (
+    <>
+      <Alert type="success" className="margin-bottom-4">
+        {t('in_person_proofing.body.prepare.alert_selected_post_office', { name: 'EASTCHESTER' })}
+      </Alert>
+      <PageHeading>{t('in_person_proofing.headings.prepare')}</PageHeading>
+
+      <p>{t('in_person_proofing.body.prepare.verify_step_about')}</p>
+
+      <ProcessList className="margin-bottom-4">
+        <ProcessListItem
+          heading={t('in_person_proofing.body.prepare.verify_step_enter_pii')}
+          headingUnstyled
+        />
+        <ProcessListItem
+          heading={t('in_person_proofing.body.prepare.verify_step_enter_phone')}
+          headingUnstyled
+        />
+      </ProcessList>
+
+      <hr className="margin-bottom-4" />
+
+      <h2>{t('in_person_proofing.body.prepare.bring_title')}</h2>
+
+      <IconList>
+        <IconListItem
+          icon="check_circle"
+          title={t('in_person_proofing.body.prepare.bring_barcode_header')}
+        >
+          <p>{t('in_person_proofing.body.prepare.bring_barcode_info')}</p>
+        </IconListItem>
+
+        <IconListItem
+          icon="check_circle"
+          title={t('in_person_proofing.body.prepare.bring_id_header')}
+        >
+          <p>{t('in_person_proofing.body.prepare.bring_id_info_acceptable')}</p>
+          <ul>
+            <li>{t('in_person_proofing.body.prepare.bring_id_info_dl')}</li>
+            <li>{t('in_person_proofing.body.prepare.bring_id_info_id_card')}</li>
+          </ul>
+          <p>{t('in_person_proofing.body.prepare.bring_id_info_no_other_forms')}</p>
+        </IconListItem>
+
+        <IconListItem
+          icon="check_circle"
+          title={t('in_person_proofing.body.prepare.bring_proof_header')}
+        >
+          <p>{t('in_person_proofing.body.prepare.bring_proof_info_acceptable')}</p>
+          <ul>
+            <li>{t('in_person_proofing.body.prepare.bring_proof_info_lease')}</li>
+            <li>{t('in_person_proofing.body.prepare.bring_proof_info_registration')}</li>
+            <li>{t('in_person_proofing.body.prepare.bring_proof_info_card')}</li>
+            <li>{t('in_person_proofing.body.prepare.bring_proof_info_policy')}</li>
+          </ul>
+        </IconListItem>
+      </IconList>
+      {inPersonURL && (
+        <div className="margin-y-5">
+          <Button href={inPersonURL} onClick={removeUnloadProtection} isBig isWide tabIndex={0}>
```

Comment:
Is the `tabIndex` necessary?

Reply:
Seems to work for me without it 🤔  You were <kbd>Tab</kbd>-ing to the link?

![image](https://user-images.githubusercontent.com/1779930/178300234-b7e7a8e6-c0d6-4dad-9c7d-ca1bc0df4ce8.png)

Reply:
Yes, tabbing; it wasn't working for me because I had "Pressing tab on a webpage highlights links, as well as form fields" toggled off in my Chrome settings. I guess it's safe to assume that anyone using assistive technology will have this turned on.

Reply:
removed the `tabIndex` in 65664ffea983cccb907d679bb7f87b1867e3bdb6